src/pages/Swap/index.tsx

```diff
@@ -133,15 +143,28 @@ export default function Swap() {
     currencies,
     inputError: swapInputError,
   } = useDerivedSwapInfo(platformOverride || undefined)
-  const { wrapType, execute: onWrap, inputError: wrapInputError } = useWrapCallback(
-    currencies[Field.INPUT],
-    currencies[Field.OUTPUT],
+
+  const isInputCurrencyNative =
+    potentialTrade && potentialTrade.inputAmount && Currency.isNative(potentialTrade?.inputAmount?.currency)
```

Comment:
I think we can remove the ternary (`? true : false`) if we cast `Currency.isNative` with `!!`

```suggestion
    potentialTrade && potentialTrade.inputAmount && !!Currency.isNative(potentialTrade?.inputAmount?.currency)
```